modules/rank-eval/src/test/java/org/elasticsearch/index/rankeval/DiscountedCumulativeGainTests.java

```diff
@@ -243,6 +246,20 @@ public void testXContentRoundtrip() throws IOException {
         }
     }
 
+    public void testXContentParsingNotLenient() throws IOException {
```

Comment:
is this testing that parsing is or is not lenient? I can't get that from the name ;) Is this part of a request or response?

Reply:
This is testing that adding random fields somewhere in the xContent throws an error. So its testing that parsing is not lenient. All these are part of the request, so I think thats the behaviour we want there, whereas we want to be lenient on the response side. I can rename all these test methods to e.g. `testRequestParsingIsNotLenient`, maybe add a short comment as well. Would that be better?

Reply:
Adding the `is` verb is good enough on my end, thanks.